features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/utils/config/GlobalRumMonitorTestConfiguration.kt

```diff
@@ -9,19 +9,33 @@ package com.datadog.android.rum.utils.config
 import com.datadog.android.core.InternalSdkCore
 import com.datadog.android.rum.GlobalRumMonitor
 import com.datadog.android.rum.RumMonitor
+import com.datadog.android.rum._RumInternalProxy
 import com.datadog.android.rum.internal.monitor.AdvancedRumMonitor
 import com.datadog.tools.unit.extensions.config.MockTestConfiguration
 import fr.xgouchet.elmyr.Forge
 import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+
+@Suppress("TestFunctionName")
+internal abstract class InternalAdvancedRumMonitor : AdvancedRumMonitor {
+    override fun _getInternal(): _RumInternalProxy? {
+        return null
+    }
+}
```

Comment:
> [!NOTE]
> Why is this part required ?

Reply:
Without explicitly overriding the `_getInternal()` method, a runtime error is thrown when executing the unit tests, saying that `AdvancedRumMonitor` does not implement the abstract method `_getInternal()`. It might be because it is marked with `@JvmSynthetic`.